examples/imagenet/main_fp16_optimizer.py

```diff
@@ -371,8 +371,6 @@ def validate(val_loader, model, criterion):
     while input is not None:
         i += 1
 
-        target = target.cuda(async=True)
-
         # compute output
```

Comment:
Remove the `target.cuda()` call, since `data_prefetcher` makes sure to push the data and target onto the GPU.